.gitignore

```diff
@@ -1,9 +1,4 @@
-*.swp
-*.beam
-.DS_Store
 .bundle
-.config
-.ruby-version
```

Comment:
Ditto. We don't care, but a lot of people don't review their diffs, and accidentally commit things we don't care about.